src/modules/Plans/actions/PlanActionsDropdown.tsx

```diff
@@ -1,17 +1,24 @@
-import React, { type Ref } from 'react';
-import { ModalHOC } from 'src/modules/Providers/modals/ModalHOC/ModalHOC';
-import { useForkliftTranslation } from 'src/utils/i18n';
+import React, { type Ref } from "react";
```

Comment:
prettier shouldn't try to change these to double quotes. Can you re-run `yarn install` and restart your workspace?